src/map.rs

```diff
@@ -539,15 +539,15 @@ pub struct Unk80809178 {
 #[derive(BinRead, Debug, Clone)]
 pub struct Unk8080917b {
     // Points to havok pre-tag
-    pub unk0: RelPointer<Unk80809121>,
+    pub unk0: RelPointer<PushSurface>,
     pub unk8: u32,
     pub unkc: u32,
     pub kind: u8,
     pub unk11: u8,
 }
 
 #[derive(BinRead, Debug, Clone)]
-pub struct Unk80809121 {
+pub struct PushSurface {
```

Comment:
Destiny structs should always start with `S`, currently except for Unk8080XXXX structs (will be addressed in new data library)

Also I dont think we should be naming these yet if we don't know what they are exactly. Just "PushSurface" seems a bit misleading right now. "SlipperySurfaceVolume" seems a bit more appropriate from what we know